Tests/UITests/UITests.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 56;
+	objectVersion = 70;
```

Comment:
We might have to jump back to a 6X project version for support the latest non-beta Xcode releases as noted in the build errors:
> [!CAUTION]
> Reason: The project ‘UITests’ cannot be opened because it is in a future Xcode project file format (70). Adjust the project format using a compatible version of Xcode to allow it to be opened by this version of Xcode.